src/app/terms/terms.component.html

```diff
@@ -25,175 +24,305 @@
+                effective
+                immediately. If you do not agree to the modified terms for a Service, you should discontinue your use
+                of
                 that Service.
-                <br><br> If there is a conflict between these terms and the additional terms, the additional terms will control
+                <br><br> If there is a conflict between these terms and the additional terms, the additional terms will
+                control
                 for that conflict.
-                <br> These terms control the relationship between susper and you. They do not create any third party beneficiary
+                <br> These terms control the relationship between susper and you. They do not create any third party
+                beneficiary
                 rights.
-                <br><br> If you do not comply with these terms, and we don’t take action right away, this doesn’t mean that
+                <br><br> If you do not comply with these terms, and we don’t take action right away, this doesn’t mean
+                that
                 we are giving up any rights that we may have (such as taking action in the future).
-                <br><br> If it turns out that a particular term is not enforceable, this will not affect any other terms.<br><br>                You agree that the laws of Can Tho, Viet Nam will apply to any disputes arising out of or relating to these
-                terms or the Services. All claims arising out of or relating to these terms or the services will be litigated
-                exclusively in the courts of Can Tho City, Viet Nam, and you and susper consent to personal jurisdiction
+                <br><br> If it turns out that a particular term is not enforceable, this will not affect any other
+                terms.<br><br> You agree that the laws of Can Tho, Viet Nam will apply to any disputes arising out of
+                or relating to these
+                terms or the Services. All claims arising out of or relating to these terms or the services will be
+                litigated
+                exclusively in the courts of Can Tho City, Viet Nam, and you and susper consent to personal
+                jurisdiction
                 in those courts.
-                <br><br><div style="margin-bottom:50px;"> For information about how to contact susper, please visit our <a class="page-link" routerLink="/contact">contact page</a>.</div>
                 <br><br>
-            </p>
+                <div style="margin-bottom:50px;"> For information about how to contact susper, please visit our <a
+                        class="page-link" routerLink="/contact">contact page</a>.</div>
+                <br><br>
+        </div>
     </div>
-</div>
 
-<footer>
-    <app-footer-navbar></app-footer-navbar>
-</footer>
+    <footer>
+        <app-footer-navbar></app-footer-navbar>
+    </footer>
```

Comment:
add EOF

Reply:
@simsausaurabh the sidebar menu can be scrolled vertically only. I didn't get what you are referring as `horizontal scrolling`. I'll add EOF btw :)

Reply:
Yes vertical scrolling is ok! Since the text size is large and width is less, it is also scrolling horizontally.